itou/www/siae_evaluations_views/forms.py

```diff
@@ -289,3 +224,71 @@ def clean(self):
             if cleaned_data.get("permanent_suspension"):
                 cleaned_data["suspension_dates"] = InclusiveDateRange(cleaned_data["permanent_suspension"])
         return cleaned_data
+
+
+class InstitutionEvaluatedSiaeNotifyStep3EvaluatedJobApplicationFormSet(forms.BaseFormSet):
+    def __init__(self, sanctions, evaluated_job_applications_to_sanction, *args, **kwargs):
```

Comment:
J'ai l'impression que le paramètre `sanctions` ne sert à rien ? On regarde juste si `if SUBSIDY_CUT in sanctions:`, chose que l'on fait déjà dans `InstitutionEvaluatedSiaeNotifyStep3View.get_forms`.